pkgs/top-level/python-packages.nix

```diff
@@ -11845,6 +11845,10 @@ in {
 
   readme_renderer = callPackage ../development/python-modules/readme_renderer { };
 
+  readlike = callPackage ../development/python-modules/readlike { };
+
+  ReParser = callPackage ../development/python-modules/ReParser { };
```

Comment:
Please use `reparser`, also for the directory name: https://github.com/NixOS/nixpkgs/pull/45822